commit ac5e508766edacb483b1321273cef31f21f1e7e8
Author: Dawid Weiss <dweiss@apache.org>
Date:   Thu Mar 1 22:28:12 2012 +0000

    Removed Thread.toString() override with complex internal logic including monitor locks.
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1295949 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/lucene/core/src/java/org/apache/lucene/index/ConcurrentMergeScheduler.java b/lucene/core/src/java/org/apache/lucene/index/ConcurrentMergeScheduler.java
index 44fe5fa..95e4c23 100644
--- a/lucene/core/src/java/org/apache/lucene/index/ConcurrentMergeScheduler.java
+++ b/lucene/core/src/java/org/apache/lucene/index/ConcurrentMergeScheduler.java
@@ -488,19 +488,6 @@ public class ConcurrentMergeScheduler extends MergeScheduler {
         }
       }
     }
-
-    @Override
-    public String toString() {
-      try {
-        MergePolicy.OneMerge merge = getRunningMerge();
-        if (merge == null) {
-          merge = startMerge;
-        }
-        return "merge thread: " + tWriter.segString(merge.segments);
-      } catch (Throwable e) {
-        return "merge thread: " + e.toString();
-      }
-    }
   }
 
   /** Called when an exception is hit in a background merge
